### ISSUE TITLE
Store bandpower windows separately to avoid reading the workspace

### ISSUE THREAD

**Issue by carlosggarcia** [xC-ell/DEScls#43]
Generating the sacc files take a long time because they require reading the workspaces to access the bandpower windows. We need to store them separately.
One idea would be to store them in a file linked to the mask, to avoid repeating them in the cl npz files.

**Comment by carlosggarcia** [comment]
Reading the DESgc-DESgc workspace for 4096 vs reading the window functions in a npz compared as:
```
13.899039s vs 0.052375s
```
and don't require much space
```
3.7M    /mnt/extraspace/gravityls_3/S8z/Cls_new_pipeline/4096_DES_eBOSS_CMB/DESgc_DESgc/windows.npz                                                                                                        
1.2G    /mnt/extraspace/gravityls_3/S8z/Cls_new_pipeline/4096_DES_eBOSS_CMB/DESgc_DESgc/w__mask_DESgc__mask_DESgc.fits
```

**Comment by carlosggarcia** [comment]
Other reason why to_sacc takes a long time to generate the sacc file might be that it initializes the mappers in order to get information about `type` and `spin`, and some mappers load the catalogs at initialization.

**Comment by damonge** [comment]
We could add a "fast" option to all mappers then, or make `spin` and `type` attributes of the class... Although doesn't it also need them for dndz?

**Comment by carlosggarcia** [comment]
Yes, you're right. They are also needed for the dndz. However, this is normally read from some file, which should take much less time than reading a catalog.

**Comment by damonge** [comment]
Yes, so maybe a "fast" option is the way to go.
